src/hooks/useSendAnswerEmail.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useState } from 'react';
+import { EmailServiceProps } from '@types';
```

Comment:
Is this import actually being used?

Reply:
And if it is, I think this should be imported with `import type { ... }`, right?

Reply:
It is being used on L:15 but I forgot about the type keyword when importing types, thank you. I updated the rest of the app to import type where possible also.